src/mapeo-project.js

```diff
@@ -124,4 +166,34 @@ export class MapeoProject {
   get field() {
     return this.#dataTypes.field
   }
+
+  /**
+   * @param {Omit<import('@mapeo/schema').ProjectValue, 'schemaName'>} settings
+   * @param {string} [versionId]
+   * @returns {Promise<import('@mapeo/schema').Project>}
+   */
+  async $setProjectSettings(settings, versionId) {
+    if (versionId) {
+      const existing = await this.#dataTypes.project.getByVersionId(versionId)
+
+      if (existing) {
+        return this.#dataTypes.project.update(existing.versionId, {
+          ...settings,
+          schemaName: 'project',
+        })
+      }
+    }
+
+    return this.#dataTypes.project[kCreateWithDocId](this.#projectId, {
+      ...settings,
+      schemaName: 'project',
+    })
+  }
+
+  /**
+   * @returns {Promise<import('@mapeo/schema').Project>}
```

Comment:
I have this returning the entire project document for now, but guessing we'd want less information to surface in general.